signingscript/task.py

```diff
@@ -190,24 +209,37 @@ def get_suitable_signing_servers(signing_servers, cert_type, signing_formats):
             zipalign_command += ['-v']
 
         zipalign_command += [_ZIP_ALIGNMENT, original_apk_location, temp_apk_location]
-        await _execute_subprocess(zipalign_command)
+        await utils._execute_subprocess(zipalign_command)
         shutil.move(temp_apk_location, abs_to)
 
     log.info('"{}" has been zip aligned'.format(abs_to))
 
 
-async def _execute_subprocess(command):
-    log.info('Running "{}"'.format(' '.join(command)))
-    subprocess = await asyncio.create_subprocess_exec(*command, stdout=PIPE, stderr=STDOUT)
-    log.info("COMMAND OUTPUT: ")
-    await utils.log_output(subprocess.stdout)
-    exitcode = await subprocess.wait()
-    log.info("exitcode {}".format(exitcode))
+# _explode_dmg {{{1
+async def _explode_dmg(context, from_):
+    """Explode a dmg and tar up its contents. Return the relative tarball path."""
+    work_dir = context.config['work_dir']
+    abs_from = os.path.join(work_dir, from_)
+    to = re.sub(r'''\.dmg$''', '.tar.gz', from_, flags=re.I)
```

Comment:
Nits:

* No need for triple-quotes, here I think.
* I believe `re.I` takes a bit of headspace. It may lead people to mentally map it to `re.IGNORECASE`, or (like I did, I admit 😃), look up the documentation to remember what that is.